src/js/letters/containers/AddressSection.jsx

```diff
@@ -144,14 +144,22 @@ export class AddressSection extends React.Component {
     let type = ADDRESS_TYPES.domestic;
     if (!['USA', 'US'].includes(address.country)) {
       type = ADDRESS_TYPES.international;
-    } else if (address.militaryStateCode) {
-      // TODO: Make sure we clear this out if a state code is selected
+    } else if (['AE', 'AA', 'AP'].includes(address.state)) {
+      // Are these ^^ constants anywhere?
```

Comment:
I haven't seen them anywhere, I think you'd have to map over the `militaryStateNames` array in `helpers` (but I think you already know this...)

Reply:
Ahh, I didn't realize it was there...not sure if I should use that or not right here, though.